swagger.yaml

```diff
@@ -3173,6 +3173,253 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+
+  /v2/infra-envs/{infra_env_id}/hosts:
+    post:
+      tags:
+        - installer
+      security:
+        - agentAuth: []
+      description: Registers a new OpenShift agent.
+      operationId: v2RegisterHost
+      parameters:
+        - in: path
+          name: infra_env_id
+          description: The InfraEnv that the agent is associated with.
+          type: string
+          format: uuid
+          required: true
+        - in: body
+          name: new-host-params
+          description: The description of the agent being registered.
+          required: true
+          schema:
+            $ref: '#/definitions/host-create-params'
```

Comment:
It can be in a laster commit, but we need to optionally pass cluster_id

Reply:
The cluster id is taken from infra-env